cottontaildb_client/cottontaildb_client.py

```diff
@@ -266,6 +266,21 @@ def get_entity_details(self, schema, entity):
         }
 
         return entity_details
+    
+    def preview_entity(self, schema, entity, limit=10):
+        """
+        Retrieves a preview of the specified entity.
+
+        @param schema: the entity's schema
+        @param entity: entity name
+        @param limit: number of rows to return
+        @return: query response message
+        """
+        schema_name = SchemaName(name=schema)
+        entity_name = EntityName(schema=schema_name, name=entity)
+        projection = Projection(op=Projection.ProjectionOperation.SELECT,
+                                elements=[Projection.ProjectionElement(expression=Expression(column=ColumnName(entity=entity_name, name = "*")))])
```

Comment:
Please format this line so it is as legible as possible with no line exceeding 120 characters in accordance with PEP 8.